.github/workflows/release.yml

```diff
@@ -76,30 +75,30 @@ jobs:
           cd .. && \
           sha256sum vela/vela-* kubectl-vela/kubectl-vela-* >> sha256-${{ steps.get_matrix.outputs.OS }}-${{ steps.get_matrix.outputs.ARCH }}.txt \
       - name: Upload Vela tar.gz
-        uses: actions/upload-release-asset@v1.0.2
+        uses: basefas/upload-release-asset-action@v1
```

Comment:
can we use an officail repo or make it a repo inside kubevela org \cc @Somefive

Reply:
BTW, why not use this https://github.com/softprops/action-gh-release as it's metioned in the archived repo as alternative.

Reply:
[action-gh-release](https://github.com/softprops/action-gh-release) will break the logic in action above.
However，copy the code to a kubevela org repo is OK.

Reply:
Great, please contribute to this repo: https://github.com/kubevela/upload-release-asset

Thanks!

Reply:
action name must be unique，so is vela-action-upload-release-asset ok？

Reply:
OK, let me update it

Reply:
https://github.com/kubevela/vela-upload-release-asset

Reply:
😂, not repo name, I edit it [here](https://kubevela/vela-upload-release-asset/pull/1/commits/5446c628727aa315f5e1fc58415cd2cd9d25b150), you will get it when you release action. 
I don't have permission, so you need merge pr, change action name in action.yml if you want.
Then, release it.
See also in kubevela/vela-upload-release-asset#1

Reply:
@basefas merged, you've been added as one of the approver of this repo and also a member of KubeVela org. Thanks for your contributions！ Hope you can continue to maintain these actions.